### ISSUE TITLE
Erroneus  message reported for ipython by the "pipenv update --dry-run" command 

### ISSUE THREAD

**Issue by pvergain** [kennethreitz/pipenv#810]
ipython 6.2.1 is installed but the "pipenv update --dry-run" command does report that the version installed is 5.5.0 and must be updated to 6.2.1.

Example:

```
C:\Tmp\test_ipython>pipenv install ipython
Creating a virtualenv for this project…
Using base prefix 'c:\\python36'
New python executable in c:\project\python_envs\test_ipython-i3RywRyq\Scripts\python.exe
Installing setuptools, pip, wheel...done.

Virtualenv location: c:\project\python_envs\test_ipython-i3RywRyq
Creating a Pipfile for this project…
Installing ipython…
Collecting ipython
  Using cached ipython-6.2.1-py3-none-any.whl
Collecting traitlets>=4.2 (from ipython)
  Using cached traitlets-4.3.2-py2.py3-none-any.whl
Requirement already satisfied: setuptools>=18.5 in c:\project\python_envs\test_ipython-i3rywryq\lib\site-packages (from ipython)
Collecting simplegeneric>0.8 (from ipython)
Collecting decorator (from ipython)
  Using cached decorator-4.1.2-py2.py3-none-any.whl
Collecting pickleshare (from ipython)
  Using cached pickleshare-0.7.4-py2.py3-none-any.whl
Collecting colorama; sys_platform == "win32" (from ipython)
  Using cached colorama-0.3.9-py2.py3-none-any.whl
Collecting prompt-toolkit<2.0.0,>=1.0.4 (from ipython)
  Using cached prompt_toolkit-1.0.15-py3-none-any.whl
Collecting jedi>=0.10 (from ipython)
  Using cached jedi-0.11.0-py2.py3-none-any.whl
Collecting pygments (from ipython)
  Using cached Pygments-2.2.0-py2.py3-none-any.whl
Collecting six (from traitlets>=4.2->ipython)
  Using cached six-1.11.0-py2.py3-none-any.whl
Collecting ipython-genutils (from traitlets>=4.2->ipython)
  Using cached ipython_genutils-0.2.0-py2.py3-none-any.whl
Collecting wcwidth (from prompt-toolkit<2.0.0,>=1.0.4->ipython)
  Using cached wcwidth-0.1.7-py2.py3-none-any.whl
Collecting parso==0.1.0 (from jedi>=0.10->ipython)
  Using cached parso-0.1.0-py2.py3-none-any.whl
Installing collected packages: six, ipython-genutils, decorator, traitlets, simplegeneric, pickleshare, colorama, wcwidth, prompt-toolkit, parso, jedi, pygments, ipython
Successfully installed colorama-0.3.9 decorator-4.1.2 ipython-6.2.1 ipython-genutils-0.2.0 jedi-0.11.0 parso-0.1.0 pickleshare-0.7.4 prompt-toolkit-1.0.15 pygments-2.2.0 simplegeneric-0.8.1 six-1.11.0 traitlets-4.3.2 wcwidth-0.1.7

Adding ipython to Pipfile's [packages]…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
Updated Pipfile.lock (081261)!

```
```
C:\Tmp\test_ipython>pipenv update --dry-run
Checking dependencies…
ipython==6.2.1 is available (5.5.0 installed)!
```
```
C:\Tmp\test_ipython>pipenv run ipython
Python 3.6.2 (v3.6.2:5fd33b5, Jul  8 2017, 04:57:36) [MSC v.1900 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]:
```

**Comment by kennethreitz** [comment]
thanks for the report

**Comment by kennethreitz** [comment]
reproduced

**Comment by kennethreitz** [comment]
this is now "fixed" — the result is now:

```
ipython==5.5.0 is available (6.2.1 installed)!
```

which is accurate. we'll work on improving the resolution later.

**Comment by BeyondEvil** [comment]
@kennethreitz I was just about to report this as an issue as I saw something similar: `py==1.4.34 is available (1.5.2 installed)!`

Is this by design? I don't get it. Why is it saying an older version is available (! yay, wat?) when I have a more recent one installed...? :)
